tensorflow/contrib/go/proto/allocation_description.pb.go

```diff
@@ -0,0 +1,129 @@
+// Code generated by protoc-gen-go.
```

Comment:
Sure, I'm going to add the generation of this files on the gen.go for "go generate"